src/main/java/me/juancarloscp52/entropy/client/VotingClient.java

```diff
@@ -106,14 +107,14 @@ public void newPoll(int voteID, List<String> events) {
         if (this.size - 1 == events.size()) {
             this.voteID = voteID;
             this.events = events;
-            this.events.add("entropy.voting.randomEvent");
+            this.events.add(Component.translatable("entropy.voting.randomEvent"));
             totalVotesCount = 0;
             votes = new int[4];
             totalVotes = new int[4];
-            sendPoll(voteID,events);
+            sendPoll(voteID, events);
             int max = 200;
-            for (String key : events){
-                int width = client.font.width(Component.literal((5) + ": ").append(Component.translatable(key)));
+            for (Component key : events){
+                int width = client.font.width(Component.literal((5) + ": ").append(key));
```

Comment:
If we're moving to components, might as well make this translatable this as well.